core/src/main/scala/org/apache/spark/sql/delta/DeltaTable.scala

```diff
@@ -85,7 +85,7 @@ object DeltaTableUtils extends PredicateHelper
    */
   def isDeltaTable(spark: SparkSession, tableName: TableIdentifier): Boolean = {
     val catalog = spark.sessionState.catalog
-    val tableIsNotTemporaryTable = !catalog.isTemporaryTable(tableName)
+    val tableIsNotTemporaryTable = !catalog.isTempView(tableName)
```

Comment:
isTemporaryTable is renamed to isTempView in Spark 3.2.0 ( https://github.com/apache/spark/pull/31295/files#diff-9dd0899e5406230aeff96654432da54f35255f6dc60eecb87264a5c508a8c826L893 )